subjects/devops/play-with-containers/audit/README.md

```diff
@@ -45,58 +44,56 @@ api-gateway-app      ... done
 user:~$
 ```
 
-###### did the student respect the architecture?
+###### Did the student architecture reflect the infrastructure enforced by the subject?
 
-###### did the infrastructure start correctly?
+###### Did the infrastructure start correctly?
 
 ##### Ask the following questions to the group or student
 
-###### What is microservices architecture?
+##### What is a microservices architecture?
 
-###### Why do we use microservices architecture?
+##### Why do we use microservices architecture?
 
-###### What is a queue and what are its benefits of it?
+##### What is a queue and what is it used for?
 
-###### What is RabbitMQ?
+##### What is RabbitMQ?
 
 ###### Did the student reply correctly to the questions?
 
-##### Verify the Dockerfiles:
+#### Verify the Dockerfiles:
 
 ###### Is there a Dockerfile for each service?
 
-###### Are all Dockerfiles based on Debian or alpine?
-
-###### There is no sensitive data in DockerFiles or other solution files, rather it's exists in the `.env`?

```

Comment:
I wonder if the question to check for `.env` was removed on purpose of by mistake?

Reply:
I guess the reason you decided to remove it is because there is a similar question before? This one is more specific, do you think it would still make sense to remove it?